pages/spicedb/concepts/watch.mdx

```diff
@@ -5,12 +5,13 @@ import { InlinePlayground } from '@/components/playground';
 
 The [Watch API] in SpiceDB enables clients to monitor changes made to [Relationships] within the system.
 
-Watch events are generated when relationships are created, touched, or deleted through the [WriteRelationships], [DeleteRelationships], or the bulk import API.
+Watch events are generated when relationships are created, touched, or deleted through the [WriteRelationships], [DeleteRelationships] or [BulkImportRelationships] APIs.
 
 [Watch API]: https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.WatchService
 [Relationships]: relationships
 [WriteRelationships]: https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.PermissionsService.WriteRelationships
 [DeleteRelationships]: https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.PermissionsService.DeleteRelationships
+[BulkImportRelationships]: https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.BulkImportRelationshipsRequest
```

Comment:
Reference [ImportBulkRelationships](https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.PermissionsService.ImportBulkRelationships) here. `BulkImport` is in the experimental API.

As for why the naming shuffle, it's because you can't have the same endpoint name in multiple services, and we wanted to retain backwards compatibility.